packages/react-native-aztec/android/build.gradle

```diff
@@ -81,11 +87,10 @@ repositories {
 
     maven { url "https://jitpack.io" }
 
-    if (!rootProject.ext.buildGutenbergFromSource) {
-        // if not building from source (where the node_modules dir is used), use a remote RN maven repo
-        def reactNativeRepo = 'https://dl.bintray.com/wordpress-mobile/react-native-mirror/'
-        println "Will use the RN maven repo at ${reactNativeRepo}"
-        maven { url reactNativeRepo }
+    if (willPublishReactNativeAztecBinary) {
+        maven { url "https://dl.bintray.com/wordpress-mobile/react-native-mirror/" }
+    } else {
+        maven { url("$rootDir/../../../node_modules/react-native/android") }
```

Comment:
Reading this from the `node_modules` in the case were the build is not run to publish the binary, means that `npm install` is a pre-requisite to running the build. 

If I understand correctly, the composite build scenario would fall into this branch, right? If so, have we got a way to help devs remember to run `npm install` when using the composite build setup? Or is this a non-issue because that's something a dev using the composite build to work with these libraries would always be doing?

Reply:
Yes, I've already added one [here](https://github.com/oguzkocer/gutenberg/blob/react-native-composite-and-binary-build-support/packages/react-native-bridge/android/settings.gradle#L27) and the build would not even make it to this line without the `node_modules` folder as the subprojects would be missing.